pulp_python/app/utils.py

```diff
@@ -150,6 +150,23 @@ def latest_content_version(content_query, version):
     return latest_content
 
 
+def json_to_dict(data):
+    """
+    Converts a JSON string into a Python dictionary.
```

Comment:
This is interesting. Why is it needed?

Reply:
the post-test was breaking because the data was already a dict:
https://github.com/pulp/pulp_python/runs/3406807006?check_suite_focus=true#step:12:645